packages/core/src/types/config/index.ts

```diff
@@ -72,6 +72,16 @@ export type Redirect = {
 };
 
 export interface LibConfig extends RsbuildConfig {
+  /**
+   * Each lib configuration has a unique identifier used to distinguish different lib configurations.
+   * By default, Rslib generates a unique identifier based on the order of lib configurations, in the format `${format}${index}`.
+   * When there is only one lib of the format, the index is empty, otherwise, it starts from 0 and increments.
+   * For example:
+   * - If only ESM and CJS formats are configured, the identifier for ESM is `esm` and for CJS is `cjs`.
+   * - If two ESM formats and one CJS format are configured, they are represented as `esm0`, `esm1`, and `cjs`.
+   * @default undefined
+   */
```

Comment:
Is it necessary to write so much detail here? Just jump to website links?

Reply:
Cleaned. We need to add link for all field in another PR, like TypeScirpt.